docs/doc-writer.php

```diff
@@ -71,10 +71,10 @@ function get_replace_position($lines) {
 }
 
 function build_documentation($docs) {
-    usort(
-        $docs,
-        function($a, $b) { return strcmp($a->methodName, $b->methodName); }
-    );
+//    usort(
```

Comment:
It's not needed, because I relying on the order the operators appear in `Observable`